dask/async.py

```diff
@@ -488,12 +488,10 @@ def fire_task():
                 task = dsk[key]
                 _execute_task(task, data)  # Re-execute locally
             else:
-                raise type(res)('\nRemote Exception:\n'
-                              + '-----------------\n'
-                              + str(res) + '\n\n'
-                              + 'Traceback:\n'
-                              + '----------\n'
-                              + tb)
+                print("Traceback:\n"
+                      "----------")
+                print(tb)
```

Comment:
This is a bit too magical for my liking though. It also is probably fragile, as it assumes that the base exception stores all attributes in `args`.